src/test/scala/org/kibanaLoadTest/helpers/Helper.scala

```diff
@@ -293,4 +295,41 @@ object Helper {
     }
     file
   }
+
+  def checkFilesExist(paths: Path*): Unit = {
+    paths.foreach(path => {
+      val file = new File(path.toString)
+      if (!file.exists()) {
+        throw new RuntimeException(s"File does not exist: ${file.getPath}")
+      }
+    })
+  }
+
+  def readArchiveFile(filePath: Path): List[String] = {
+    checkFilesExist(filePath)
+    val inputStream = if (filePath.toString.endsWith(".gz")) {
+      new InputStreamReader(
+        new GZIPInputStream(new FileInputStream(filePath.toFile)),
+        "UTF-8"
+      )
+    } else if (filePath.toString.endsWith(".json")) {
+      new InputStreamReader(new FileInputStream(filePath.toFile))
+    } else throw new RuntimeException(s"Cannot parse the file: $filePath")
+    val br = new BufferedReader(inputStream)
+    var strLine = Option(br.readLine())
+    var jsonString = ""
+    val jsonStringBuffer = new ArrayBuffer[String]()
+    while (strLine.isDefined) {
+      val nextLine = Option(br.readLine())
+      if (nextLine.isEmpty || nextLine.get.length == 0) {
+        jsonString += strLine.get
```

Comment:
At least in Java the idiomatic way is to use a `StringBuilder` instead of concatenating strings directly. I'm not sure whether it's done differently in Scala.